helmfile/charts/grafana-ops/values.yaml

```diff
@@ -103,6 +103,6 @@ dashboards:
   welcome:
     enabled: true
     user_visible: true
-  ciskubernetesbenchmark:
+  trivy-operator:
```

Comment:
What is the plan for the old dashboards, are you planning on deleting them or adapting them to the new metrics?

Reply:
Yeah, my plan currently is that i may use this: https://github.com/elastisys/ck8s-ops/pull/3734#discussion_r1155999742 to create a new dashboard in-place of the old one! :+1:

Reply:
This can now be viewed in Grafana!